<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QWidget" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="container" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>70</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="button_container">
       <property name="spacing">
        <number>100</number>
       </property>
       <item>
        <widget class="QPushButton" name="ui_new_game_button">
         <property name="accessibleName">
          <string/>
         </property>
         <property name="text">
          <string>New Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_load_game_button">
         <property name="text">
          <string>Load Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_exit_game_button">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
